From: Thomas Klute <thomas2.klute@uni-dortmund.de>
Date: Sun, 19 Feb 2017 18:57:56 +0100
Subject: Do not treat warnings about deprecated declarations as errors

GnuTLS has declared OpenPGP support as deprecated in version
3.5.9. Treating deprecation warnings as errors causes the build to
fail with this version, so exempt them from "-Werror" until OpenPGP
support is removed from mod_gnutls.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index cc3e8ae..3335773 100644
--- a/configure.ac
+++ b/configure.ac
@@ -64,7 +64,7 @@ AC_ARG_ENABLE(strict,
 
 STRICT_CFLAGS=""
 if test "$use_strict" != "no"; then
-        STRICT_CFLAGS="-Wall -Werror -Wextra"
+	STRICT_CFLAGS="-Wall -Werror -Wextra -Wno-error=deprecated-declarations"
 fi
 
 AC_MSG_CHECKING([whether to enable SRP functionality])
